chart/values.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "properties": {
+    "agentTLSMode": {
+      "type": ["string", "null"],
+      "enum": ["strict", "system-store", null],
+      "description": "agentTLSMode must be 'strict' or 'system-store' or null (undefined)"
+    },
+    "extraEnv": {
```

Comment:
I would avoid adding schema verification for this value. If you add something to this existing field you need to consider backward comparability and breaking changes, which I think is unnecessary/unhelpful for your use case.

Reply:
I thought you had mentioned us needing to add some sort of validation so people are less likely to incorrectly configure things. However that was before my unexpected time off and I was probably confused.  I'll remove this.